ddtrace/opentracer/tracer.py

```diff
@@ -71,35 +75,123 @@ def __init__(self, service_name=None, config=None, scope_manager=None):
 
     @property
     def scope_manager(self):
-        """"""
+        """Returns the scope manager being used by this tracer."""
         return self._scope_manager
 
     @property
     def active_span(self):
-        """"""
-        pass
+        """Gets the active span from the scope manager or none if it does not exist."""
+        scope = self._scope_manager.active
+        return scope.span if scope else None
```

Comment:
do we need to replicate this logic? is already available in OpenTracing API: https://github.com/opentracing/opentracing-python/blob/v2.0.0/opentracing/tracer.py#L57-L67

Relying as much in their API, means having good integration tests (that will fail) if the API behavior is changed

Reply:
Ah! Didn't see this. 👍